src/TheLow.sol

```diff
@@ -1,49 +1,163 @@
-// SPDX-License-Identifier: UNLICENSED
-pragma solidity ^0.8.13;
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.13;
 
 import {ERC721} from "solmate/tokens/ERC721.sol";
+import {Owned} from "solmate/auth/Owned.sol";
+import "./utils.sol";
 
-contract TheLow is ERC721 {
+/// @notice
+contract TheLow is ERC721, Owned {
     //
+    event SupplyUpdated(uint8 indexed newSupply);
+    // event MetadataUpdated(string uri);
+    // event MetadataFrozen();
 
-    string[] internal _tierURIs;
-    mapping(uint256 => string) _tokenURIs;
+    uint8 public constant MAX_SUPPLY = 222;
+    uint8 public totalSupply = 222;
+
+    struct Tier {
+        string name;
+        string rarity;
+        string image_cid;
+        string animation_cid;
+        string animation_hash;
+        uint8 portion;  // How many tokenIds, out of 222, of this type will be created
```

Comment:
Not blocking, but let's eventually convert this to @param doc as part of NatSpec on this struct.